protobuf/services/worker.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 
-import "requests/header.proto";
-import "requests/register_function.proto";
+import "endpoints/header.proto";
+import "endpoints/register_function.proto";
 
 service WorkerService {
   rpc Initiate(HeaderRequest) returns (HeaderResponse);
```

Comment:
this has a bit of atm machine vibes

Reply:
lol is that bad? I couldn't come up with another verb that made sense here. Other than like `sendHeaderRequest` which i was trying to avoid

Reply:
oh, i was talking about the WorkerService service def. `initiate` seems fine
i suppose you could go down the route of `checkCompatibility` or smth

Reply:
ah i see. let's stick with workerservice because `WorkerStub` don't make sense to me. And the header call will do more than just check compatibility, it's just a placeholder for any future metadata we may want/need to send